### DISCUSSION TITLE
Video Tutorials for IDOM

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#658] [Ideas]
Another way to help people better understand how to use IDOM are video tutorials. Ideally one could have a set of video tutorials that correspond to the high level sections of documentation (getting started, creating interfaces, etc.).

For example, I gave [a talk on IDOM](https://www.youtube.com/watch?v=0UVVSRy7kgE&t=1s) some time ago that I plan to re-record due to a bug in a demo.

cc: @Archmonger

**Comment by Archmonger** [comment]
Since this is hugely beneficial to both Conreq and IDOM adoption, I'll be looking to tackle this sometime this year.
